webpack.config.js

```diff
@@ -18,19 +19,38 @@ module.exports = () => {
       rules: [
         {
           test: /\.tsx?$/,
-          use: 'ts-loader',
+          use: {
+            loader: 'ts-loader', options: {
+              configFile: 'tsconfig.json'
+            }
+          },
           exclude: /node_modules/,
         },
         {
           test: /\.css$/i,
-          use: [{ loader: 'style-loader' }, { loader: 'css-loader' }],
+          use: ['style-loader', 'css-loader'],
         },
+        {
+          test: /\.module.s[ac]ss$/i,
+          use: [
+            process.env.NODE_ENV !== 'production'
```

Comment:
Будет ли такое, что на проде одни стили, а на деве другие?

Reply:
Recommended
For production builds it's recommended to extract the CSS from your bundle being able to use parallel loading of CSS/JS resources later on. This can be achieved by using the mini-css-extract-plugin, because it creates separate css files. For development mode (including webpack-dev-server) you can use [style-loader](https://webpack.js.org/loaders/style-loader/), because it injects CSS into the DOM using multiple and works faster.